test/fixtures/linter/rules/Directives/Directives.js

```diff
@@ -0,0 +1,97 @@
+sap.ui.define([
+	"sap/m/Button", "sap/m/DateTimeInput", "sap/base/util/includes", "sap/ui/Device", "sap/ui/core/library", "sap/ui/generic/app/navigation/service/NavigationHandler",
+	"sap/ui/table/Table", "sap/ui/table/plugins/MultiSelectionPlugin", "sap/ui/core/Configuration", "sap/m/library"
+], function(Button, DateTimeInput, includes, Device, coreLib, NavigationHandler, Table, MultiSelectionPlugin, Configuration, mobileLib) {
+
+	// ui5linter-disable-next-line ui5-linter-no-deprecated-api
```

Comment:
i am in favor of using `ui5lint-` instead of `ui5linter`.

Why?
-  shorter, less chars
- same as `ui5lint` command
- follows `eslint-` comments

Reply:
Fully agree 👍  This commit was created before that discussion, I forgot to update it.

Reply:
done